launch_ros_sandbox/actions/load_docker_nodes.py

```diff
@@ -161,7 +162,7 @@ def _load_nodes_in_docker(
 
     def _handle_logs(
         self,
-        log_generator
+        log_generator: GeneratorType
```

Comment:
Maybe `Generator[Any]` would be more consistent?

Reply:
Type hints from `typing` don't allow for `isinstance` checking, and I can't perform a generic "try to iterate and catch TypeError", because a `bytes` is iterable, but we wouldn't want to iterate over it.

I could say `Generator[Any]` here and check for `isinstance(thing, GeneratorType)` below - does that sound right?

Reply:
I guess in this case you current implementation LGTM, it's just slightly confusing, but Python is confusing, not this PR :) Your alternative suggestion would also be OK to me.